client/app/bundles/momentCard/components/MomentCardSettings.jsx

```diff
@@ -0,0 +1,32 @@
+// @flow
+import React from 'react';
+import MomentCardViewers from './MomentCardViewers';
+import css from './MomentCard.scss';
+
+type MomentCardSettingsState = {};
+
+type MomentCardSettingsProp = {
+  cardType: string,
+};
+
+export default class MomentCardSettings extends
+  React.Component<MomentCardSettingsProp, MomentCardSettingsState> {
```

Comment:
You can omit MomentCardViewersState as the second parameter (it's optional).

On the following lines, you can omit `props: MomentCardViewersProp` and `state: MomentCardViewersState` because they should already be defined as part of React.Component< MomentCardViewersProp>.